### ISSUE TITLE
Improve the database configuration

### ISSUE THREAD

**Issue by gabrielecker** [animapp-dev/animapp-server#4]
We must use environment variables instead of raw strings and then allow local configuration for anyone who wants to help

**Comment by mblodorn** [comment]
You are right, I'll try to find extra time to do that. It's also a security issue that we have right now.
